src/clients/python/CMakeLists.txt

```diff
@@ -40,12 +43,29 @@ if(NOT WIN32)
 # libcshm.so
 #
 add_library(cshm SHARED shared_memory/shared_memory.cc)
+if(${TRTIS_ENABLE_GPU})
```

Comment:
Why is this needed?

Reply:
The shared memory handle is now shared (single handle representation) between the two libraries. This means that when GPU is enabled the shared memory library will also need to include cuda_runtime to be able to compile successfully.